### PR TITLE
Ensure our local flatbuffers.h is included in preference to system variants

### PR DESCRIPTION
**Pull request by steven-johnson**
The local version might be too old for TFLite

### CODE REVIEW

**Comment by alexreinking** [comment]
Needs a comment but otherwise LGTM

**Comment by steven-johnson** [comment]
PTAL

File: apps/hannk/tflite/CMakeLists.txt [hunk 1]
```diff
@@ -13,6 +13,11 @@ set(TFLITE_ENABLE_RUY OFF)
 set(TFLITE_ENABLE_XNNPACK OFF)
 
 # Also have to disable some stuff dragged in by TFLite
+if(POLICY CMP0077)
+  # This won't do the trick -- we need to use CMAKE_POLICY_DEFAULT_CMP0077, alas
+  # cmake_policy(SET CMP0077 NEW)
+  set(CMAKE_POLICY_DEFAULT_CMP0077 NEW)
+endif()
```
Comment (alexreinking) [inline apps/hannk/tflite/CMakeLists.txt:20]: No need to test (`if`) for the policy before setting it because the minimum is 3.16, which means the policy exists. This line should also be moved to the top-level, beneath the `project()` call in `apps/hannk/CMakeLists.txt`.

**Review by alexreinking — CHANGES_REQUESTED** [review]

File: apps/hannk/tflite/CMakeLists.txt (hunk 1 above)
Comment (steven-johnson) [reply apps/hannk/tflite/CMakeLists.txt:20]: done

**Review by alexreinking — APPROVED** [review]
